verilog/preprocessor/verilog_preprocess.cc

```diff
@@ -53,6 +54,16 @@ VerilogPreprocess::VerilogPreprocess(const Config& config) : config_(config) {
       BranchBlock(true, true, verible::TokenInfo::EOFToken()));
 }
```

Comment:
Instead of repeating the constructor init and body, could you just use a delegating constructor that calls the other one ?

```c++
VerilogPreprocess::VerilogPreprocess(const Config& config) : VerilogPreprocess(config, nullptr) {}
```